src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -66,16 +66,16 @@ const defaultProps = {
 function OptionRowLHN(props) {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
+    const [isMounted, setIsMounted] = useState(false);
+
     useEffect(() => {
-        const updateReportDraft = () => {
-            if (!optionItem || optionItem.hasDraftComment || !props.comment || props.comment.length <= 0 || props.isFocused) {
-                return;
-            }
-            Report.setReportWithDraft(props.reportID, true);
-        };
+        if (isMounted || !optionItem) return;
+        setIsMounted(true);
```

Comment:
@mkarkachov I think you meant to call the `useEffect` once, but creating a new state is over-engineered. Can we pass an empty array for the second argument of `useEffect` and add `// eslint-disable-next-line react-hooks/exhaustive-deps` to disable the lint?

Reply:
@mollfpr sure